PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalInternalClient.java

```diff
@@ -59,6 +62,10 @@ void sendRequest(final Context context, final PayPalRequest payPalRequest,
                         ? SETUP_BILLING_AGREEMENT_ENDPOINT : CREATE_SINGLE_PAYMENT_ENDPOINT;
                 String url = String.format("/v1/%s", endpoint);
                 String appLinkReturn = isBillingAgreement ? appLink : null;
+                
+                final String linkType = (isBillingAgreement &&
```

Comment:
Nitpick - on iOS too, I think we want to make this linkType an enum at some point. We could do that here instead of switching on a String.

Reply:
Added: [8a76214](https://github.com/braintree/braintree_android/pull/1092/commits/8a762142cfb7d5940f49867c7afcbb6b92ee6142)